pyccel/codegen/printing/cwrappercode.py

```diff
@@ -26,25 +26,50 @@
 class CWrapperCodePrinter(CCodePrinter):
     def __init__(self, parser, settings={}):
         CCodePrinter.__init__(self, parser,settings)
+        self._cast_functions_set = set()
 
     def get_new_name(self, used_names, requested_name):
         if requested_name not in used_names:
             used_names.add(requested_name)
             return requested_name
         else:
-            return create_incremented_string(used_names, prefix=requested_name)
-
-    def get_PyArgParseType(self, dtype):
-        #TODO: Depends on type, rank, etc
-        if dtype is NativeBool():
-            return NativeInteger(), lambda arg, tmp: Assign(arg, Bool(tmp))
-        return dtype, None
-
-    def get_PyBuildeValue(self, dtype):
-        #TODO: Depends on type, rank, etc
-        if dtype is NativeBool():
-            return NativeInteger(), lambda arg, tmp: Assign(arg, Bool(tmp))
-        return dtype, None
+            incremented_name, _ = create_incremented_string(used_names, prefix=requested_name)
```

Comment:
By the way to get rid of Codacy's warning about unused variables you have to name your variable `_` . It is **very** rare that we need to disable codacy's warnings, especially outside of the `tests/` folder. If you're not sure how to get of a warning, ask